recipes/r-loader/meta.yaml

```diff
@@ -0,0 +1,79 @@
+{% set version = '1.7.0' %}
```

Comment:
Latest version seems to 1.7.1. Any reason for that older version?

Reply:
Yes, we have research artifacts depending on this version and we want to publish it, before to bump version.